src/hpp/corbaserver/manipulation/constraint_graph.py

```diff
@@ -512,6 +512,21 @@ def  applyNodeConstraints (self, node,  input) :
     #
     #  Compute a configuration in the destination node of the edge,
     #  reachable from qFrom.
+    def applyEdgeLeafConstraints (self, edge, qfrom, input) :
```

Comment:
I think the documentation is not correct here. The applied constraints do not contain the constraints of the destination state, do they ?

Reply:
Fixed by f85bdfe.